### PR TITLE
Form & Styling

### PR DESCRIPTION
**Pull request by iva585**

### CODE REVIEW

File: src/app/api/storage/recipes.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+};
+
+const setItems = (items: Recipe[]): void => {
+  localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify(items));
+};
+
+const addRecipeItemToStorage = (newRecipeData: Omit<Recipe, 'id'>): Recipe => {
+  const items = getAllRecipeItemsFromStorage();
+
+  const sortedbyIds = [...items].sort((a, b) => a.id - b.id);
```
Comment (julialoeschel) [inline src/app/api/storage/recipes.ts:18]: nice

File: src/app/components/Headers/HeaderRecipes.tsx [hunk 2]
```diff
@@ -8,8 +8,8 @@ export default () => {
       <Paper
         sx={{
           display: 'flex',
-          width: '250px',
-          height: '2.2rem',
+          width: '230px',
+          height: '2rem',
```
Comment (julialoeschel) [inline src/app/components/Headers/HeaderRecipes.tsx:12]: you mixed em and px here, pople tole me not to do that :)

File: src/app/pages/RecipeDetail/RecipeDetail.tsx [hunk 3]
```diff
@@ -8,20 +8,74 @@ import EditOutlinedIcon from '@mui/icons-material/EditOutlined';
...
+    'italian',
+  ],
+};
+
+type RecipeParams = 'recipeId';
 
 export default (): JSX.Element => {
   const [editing, setEditing] = useState<boolean>(false);
+  const { recipeId } = useParams<RecipeParams>();
+  console.log('recipeId: ', recipeId);
```
Comment (derjubu) [inline src/app/pages/RecipeDetail/RecipeDetail.tsx:64]: You can remove this in production

**Review by derjubu — APPROVED** [review]
LGTM 💪

**Pull request merged**